addons/account/models/account_payment_method.py

```diff
@@ -125,7 +125,7 @@ def _ensure_unique_name_for_journal(self):
             FROM account_payment_method_line apml
             JOIN account_payment_method apm ON apml.payment_method_id = apm.id
             GROUP BY apml.name, journal_id, apm.payment_type
-            HAVING count(apml.id) > 1
+            HAVING count(apml.id) > 1 AND journal_id IS NOT NULL
```

Comment:
This new condition should be in the `WHERE` clause. And please add the table alias on the field too.

Reply:
Done, thank you!